### PR TITLE
Bob stolzberg test

### PR DESCRIPTION
**Pull request by BobStolzberg**
Updated the Vormetric KB article

### CODE REVIEW

File: Ecosystem Partners/getting-started-with-vormetric-dsm.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+-----
+
+If you are interested in seeing this type of Partner Template deployment as an automated feature in the future, please share your input with us at [features@centurylinkcloud.com](mailto:features@centurylinkcloud.com)
+
+### Pricing
+The costs associated with this deployment is $195/hr for the CenturyLink Cloud Service Tasks.  There are no Vormetric license costs or additional fees bundled in.  Service Task Fees are available here: http://www.centurylinkcloud.com/service-tasks#Pricing
+
+### Frequently Asked Questions
+Frequently Asked Questions:
```
Comment (nathanyoung) [inline Ecosystem Partners/getting-started-with-vormetric-dsm.md]: Format for FAQ seems messed up. The second "Frequently Asked Questions:" is redundant.

File: Ecosystem Partners/getting-started-with-vormetric-dsm.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+- #### Where do I obtain my Vormetric License or entitlements?
+- Existing CenturyLink Enterprise Customers can contact their Account Representative for help obtaining a Vormetric license, or contact Vormetric directly:
+-   Contact Vormetric Support via telephone: (877) 267-3247
+-   Contact Vormetric via their website: http://www.Vormetric.com
+-   Email Sales via support@vormetric.com
+
+
+- #### Who should I contact for support?
+* For issues related to deploying the Centerity Blueprint on CenturyLink Cloud, Licensing or Accessing the deployed software, please visit the Centerity Support Website TBD
+* For issues related to cloud infrastructure (VM’s, network, etc), please open a ticket using the CenturyLink Cloud Support Process.
```
Comment (nathanyoung) [inline Ecosystem Partners/getting-started-with-vormetric-dsm.md]: The FAQ questions and answers don't need to be in a list format.

**Comment by BobStolzberg** [comment]
Made changes to Vormetric KB article.

Added new Bitnami KB articles.

**Comment by BobStolzberg** [comment]
Added How to publish Partner Template kb

**Pull request merged**